source/community/reactnative/src/components/containers/container.js

```diff
@@ -51,9 +52,21 @@ export class Container extends React.Component {
 
 	internalRenderer = () => {
 		const payload = this.payload;
+		let containerStyle;
+		let minHeight = payload.minHeight;
+		if (minHeight) {
+			if (typeof (minHeight) == 'string') {
+				const lastIndex = minHeight.lastIndexOf('px');
+				minHeight = lastIndex != -1 ? parseInt(minHeight.substring(0, lastIndex)) : minHeight
```

Comment:
The value "ABCpx" will evaluate to minHeight == NaN, which I think should be explicitly handled as an invalid minHeight value.

Reply:
Thanks, @dclaux. Will evaluate that pattern also.

Reply:
@dclaux - Created a method to handle the above pattern. Please provide your comment on this.